### DISCUSSION TITLE
how do i send samsung protocol, address code, and command code with this library?

### DISCUSSION THREAD

**Discussion by quinman24** [ukw100/IRMP#96] [Q&A]
I'm having issues fingering out how to send my samsung protocol, address code and command code with your library. i thot you just do this "sendSAMSUNG(IR_SEND_PIN, 0x20, 0x24, 0);" where sendSAMSUNG sends the protocol samsung and the 0x20 is the address, and the 0x24 is the command to send with that protocol. but it doesn't work.

so how would i send samsung data with my IR transmitter?

**Comment by ArminJo** [comment]
Wrong library!!!
